### DISCUSSION TITLE
UHC File Downloads Slower Than Before?

### DISCUSSION THREAD

**Discussion by rgiljohann** [CMSgov/price-transparency-guide#746] [Q&A]
Is anyone noticing slower download times with United files? We have been downloading files for a while, and now our processing is taking a lot longer than normal. Even trying to download a file to the server takes longer than it seems it should. 

I know their storage instance has changed as below contains an example URL from a past month, and an example file from the current month. 

Old Month URL:
https://uhc-tic-mrf.azureedge.net/public-mrf/2024-01-01/2024-01-01_United-HealthCare-Services--Inc-_Third-Party-Administrator_PPO-70_I3_in-network-rates.json.gz

Current Month URL:
https://mrfstorageprod.blob.core.windows.net/public-mrf/2024-05-01/2024-05-01_UnitedHealthcare-of-Illinois--Inc-_Insurer_Choice-HMO_560_in-network-rates.json.gz?sp=r&st=2024-03-26T04:49:21Z&se=2025-03-25T12:49:21Z&spr=https&sv=2022-11-02&sr=c&sig=M0sm1qeV6LULQexjwJYsuupRKv1UpgsQLzpfLtZbzkk%3D

**Comment by jamesonnetworks-clearcost** [comment]
There seem to be other issues with downloads from UHC as well. There is another discussion ongoing at https://github.com/CMSgov/price-transparency-guide/discussions/752
